internal/resources/cluster/resource_cluster.go

```diff
@@ -27,6 +31,12 @@ import (
 	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/common"
 )
 
+const (
+	unknown = iota
```

Comment:
is this really needed ? can this be avoided ?

Reply:
For the switch case you asked to use instead of if conditions.

Reply:
Switch can be implemented without this iota and this is not needed

Reply:
Can you suggest the switching parameter passing the input kubeconfig value to k8sclient?

Reply:
you can use 
switch true{

}


or just switch {

}

This will solve ur use case

Reply:
Removed it and added switch case